src/fragments/Graph.react.js

```diff
@@ -154,11 +215,72 @@ class PlotlyGraph extends Component {
         clearExtendData();
     }
 
-    graphResize() {
+    getConfig(config, responsive) {
+        return mergeDeepRight(config, this.getConfigOverride(responsive));
+    }
+
+    getLayout(layout, responsive) {
+        if (!layout) {
+            return layout;
+        }
+
+        return mergeDeepRight(layout, this.getLayoutOverride(responsive));
+    }
+
+    getConfigOverride(responsive) {
+        switch (responsive) {
+            case false:
+                return UNRESPONSIVE_CONFIG;
+            case true:
+                return RESPONSIVE_CONFIG;
+            default:
+                return AUTO_CONFIG;
+        }
+    }
+
+    getLayoutOverride(responsive) {
+        switch (responsive) {
+            case false:
+                return UNRESPONSIVE_LAYOUT;
+            case true:
+                return RESPONSIVE_LAYOUT;
+            default:
+                return AUTO_LAYOUT;
+        }
+    }
+
+    isResponsive(props) {
+        const {config, figure, responsive} = props;
+
+        if (type(responsive) === 'Boolean') {
+            return responsive;
+        }
+
+        return Boolean(
+            config.responsive &&
+                (!figure.layout ||
+                    ((figure.layout.autosize ||
+                        isNil(figure.layout.autosize)) &&
+                        (isNil(figure.layout.height) ||
+                            isNil(figure.layout.width))))
+        );
+    }
+
+    async graphResize(force = false) {
+        if (!force && !this.isResponsive(this.props)) {
+            return;
+        }
+
         const gd = this.gd.current;
-        if (gd) {
-            Plotly.Plots.resize(gd);
+        if (!gd) {
+            return;
         }
+
+        gd.classList.add('dash-graph-resizing');
+
+        Plotly.Plots.resize(gd)
+            .catch(() => {})
+            .finally(() => gd.classList.remove('dash-graph-resizing'));
```

Comment:
looks like IE11 doesn't support `finally` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/finally
anyway you've already caught, so can't you just `.then`?

Reply:
Promise.finally gets polyfilled by https://github.com/zloirock/core-js/blob/master/packages/core-js/modules/es.promise.finally.js